health-monitors/gpu-health-monitor/gpu_health_monitor/tests/test_platform_connector/test_platform_connector.py

```
@@ -1316,3 +1317,162 @@ def test_event_retry_and_cache_cleanup_when_platform_connector_down(self) -> Non
+        with tempfile.NamedTemporaryFile(mode="w", delete=False, suffix="_test_state") as f:
+            f.write("test_boot_id")
+            state_file_path = f.name
+
+        # Shorten the retry budget so a regression makes the test slow.
+        original_max_retries = platform_connector.MAX_RETRIES
+        original_initial_delay = platform_connector.INITIAL_DELAY
+        platform_connector.MAX_RETRIES = 3
+        platform_connector.INITIAL_DELAY = 1
+
+        try:
+            exit = Event()
```

Ruff (0.15.12)

[error] Variable `exit` is shadowing a Python builtin

(A001)